src/odb/include/odb/geom.h

```diff
@@ -331,11 +331,11 @@ class Rect : public GeomShape
   void mergeInit();
 
   // Indicates if the box has a negative width or height
-  bool isInverted();
+  bool isInverted() const;
 
-  uint minDXDY();
-  uint maxDXDY();
-  int getDir();
+  uint minDXDY() const;
+  uint maxDXDY() const;
```

Comment:
@maliberty how do you feel about returning `uint`s? Technically it is the safe option to avoid overflow, but it gets compared to `int`s quite often which creates warnings.

Reply:
I would use ints.  unsigned isn't worth the trouble.

Reply:
I tried doing a quick conversion and it quickly turned hairy because it is used in derived classes, other places and functions, etc. I'm just going to do `uint`-->`int` casting in this PR and then save that for a different PR